app/helpers/content_view_helper.rb

```diff
@@ -0,0 +1,19 @@
+module ContentViewHelper
+  def full_title_for(content)
+    "#{t(content_type_number(content), number: content.number)}: #{content.name}"
+  end
+
+  def short_title_for(content)
+    content.name
+  end
+
+  private
+
+  def content_type_number(content)
+    "#{content_type(content)}_number"
+  end
+
+  def content_type(content)
+    content.is_a?(Exercise) ? "exercise" : content.class.name.downcase
+  end
```

Comment:
Me trying to get line 17 approved

![image](https://user-images.githubusercontent.com/11304439/109497388-bcf48780-7a70-11eb-88b7-b61eac566ec1.png)

Reply:
It's not that bad! Just check if model_name already does that

Reply:
That's great, it does work!

Reply:
![image](https://user-images.githubusercontent.com/11860076/109517911-832f7b00-7a88-11eb-815d-35bd8488302b.png)